modules/mod_oembed/support/oembed_client.erl

```diff
@@ -19,140 +19,78 @@
 -module(oembed_client).
 
 -author("Arjan Scherpenisse <arjan@scherpenisse.net>").
--behaviour(gen_server).
```

Comment:
Gen_server asynchronicity was dropped. Will this impact performance in any way or where the `handle_casts` not useful anyway?